client/src/components/Sponsors/Sponsors.js

```diff
@@ -1,12 +1,34 @@
 import React from 'react';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import './Sponsors.css';
+import SponserCard from './SponsorsCard';
 
 function Sponsors() {
   return (
-    <div className="sponsors-component">
-      <h1>This is the Sponsors page.</h1>
+    <>
+     <div className="container-fluid col-10 mx-auto">
+    <div className="my-5">
+    <br/><br/> <br/> <br/> <br/>
+     <h1 className="text-center">Our Partners</h1>
     </div>
+    <div className="row">
+    <SponserCard
+      title = "Partner"
+    />
+   <SponserCard
+       title = "Partner"
+   />
+   <SponserCard
+       title = "Partner"
+   />
+   <SponserCard
+       title = "Partner"
+   />
+
+    </div>
+    </div>
+
+    </>
```

Comment:
Please make the indentation clear.

Reply:
are there any vscode config files for indentations stuff?

Reply:
Yes. That sounds like a good issue to me! 😄